daffodil-cli/build.sbt

```diff
@@ -42,6 +42,18 @@ mappings in Universal ++= Seq(
   baseDirectory.value / "README.md" -> "README.md",
 )
 
+mappings in Universal ++= Seq(
+  baseDirectory.value / ".." / "daffodil-runtime2" / "src" / "main" / "c" / "common_runtime.h" -> "include/common_runtime.h",
+  baseDirectory.value / ".." / "daffodil-runtime2" / "src" / "main" / "c" / "daffodil_argp.h" -> "include/daffodil_argp.h",
+  baseDirectory.value / ".." / "daffodil-runtime2" / "src" / "main" / "c" / "stack.h" -> "include/stack.h",
+  baseDirectory.value / ".." / "daffodil-runtime2" / "src" / "main" / "c" / "xml_reader.h" -> "include/xml_reader.h",
+  baseDirectory.value / ".." / "daffodil-runtime2" / "src" / "main" / "c" / "xml_writer.h" -> "include/xml_writer.h",
+)
+
+mappings in Universal ++= Seq(
+  baseDirectory.value / ".." / "daffodil-runtime2" / "target" / "streams" / "compile" / "ccTargetMap" / "_global" / "streams" / "compile" / "sbtcc.Library" / "libruntime2.a" -> "lib/libruntime2.a",
```

Comment:
The mappings in Universal determines how we package the convenience binaries that we release in zip/tar/rpm form. So this is saying to install the header files from the runtime2 directory into ``include/`` and the compiled static library into ``lib/``. You're right that explicitly mentioning target is uncommon in an sbt file. I would expect the cc plugin to make some property available to get access to the statically lib. Also, like above, we should be able to use a glob for these headers.

Reply:
I think this is something that has to be done in the main ``build.sbt`` config. I dont't think you can access settings in a sibling/parent project. The ``..`` path stuff is sort of a hack to get access files in other subprojects since sbt doesn't care if you directly access files. But it can't be done with settings. So the main build.sbt probably needs something along the lines of this:

```scala
lazy val cli  = Project("daffodil-cli", file("daffodil-cli")).configs(IntegrationTest)
                  ...
                  .settings(mappings in Universal ++= (Compile / runtime2 / ccLinkLibraries)) 
```

Not sure exactly the syntax, but it's going to be somethign like that